lib/OpenQA/WebAPI.pm

```diff
@@ -44,6 +44,7 @@ sub _read_config {
             suse_mirror   => undef,
             scm           => undef,
             hsts          => 365,
+            audit_enabled => '1',
```

Comment:
I don't know, I thought I saw somewhere else in config that numbers were quoted, but I'll redo that to int.